app/services/journals/create_service.rb

```diff
@@ -161,9 +163,9 @@ def create_journal_sql(predecessor, notes)
         ), changes AS (
           #{select_changed_sql}
         ), insert_data AS (
-          #{insert_data_sql(notes, predecessor)}
+          #{insert_data_sql(predecessor, notes, cause)}
```

Comment:
All other methods here used the order `predecessor, notes` so I changed the signature of this method to follow